omero_marshal/decode/decoders/screen.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import AnnotatableDecoder
+from omero.model import ScreenI
+
+
+class Screen201501Decoder(AnnotatableDecoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Screen'
+
+    OMERO_CLASS = ScreenI
+
+    def decode(self, data):
+        v = super(Screen201501Decoder, self).decode(data)
+        self.set_property(v, 'name', data.get('Name'))
+        self.set_property(v, 'description', data.get('Description'))
+        self.set_property(v, 'protocolDescription',
+                          data.get('ProtocolDescription'))
+        self.set_property(v, 'protocolIdentifier',
+                          data.get('ProtocolIdentifier'))
+        self.set_property(v, 'reagentSetDescription',
+                          data.get('ReagentSetDescription'))
+        self.set_property(v, 'reagentSetIdentifier',
+                          data.get('ReagentSetIdentifier'))
+        self.set_property(v, 'type', data.get('Type'))
+        for plate in data.get('Plates', list()):
```

Comment:
I was just replicating what I found in Project decoder:
```for dataset in data.get('Datasets', list()): ```
If 'Plates' is missing, we will get an empty list so that everything inside the ```for``` block will be skipped.
Will this result in all existing Plates getting unlinked when we save to OMERO? If so then it seems that we'll always need to have Plates loaded when saving? Or do we need to set the ```isPlateLinksLoaded``` flag to indicate that Screen-Plate links should not be broken?